frontend/public/components/modals/create-namespace-modal.jsx

```diff
@@ -108,108 +109,115 @@ const CreateNamespaceModal = connect(
-                <div className="modal-body__field ">
-                  <Dropdown
-                    selectedKey={this.state.np}
-                    items={defaultNetworkPolicies}
-                    dropDownClassName="dropdown--full-width"
-                    id="dropdown-selectbox"
-                    onChange={(np) => this.setState({ np })}
-                  />
-                </div>
-              </div>
-            )}
-          </ModalBody>
-          <ModalSubmitFooter
-            errorMessage={this.state.errorMessage}
-            inProgress={this.state.inProgress}
-            submitText="Create"
-            cancel={this.props.cancel.bind(this)}
-          />
-        </form>
+                {this.props.createProject && (
+                  <div className="form-group">
+                    <label htmlFor="input-display-name" className="control-label">
+                      {t('public~Display Name')}
+                    </label>
+                    <div className="modal-body__field">
+                      <input
+                        id="input-display-name"
+                        name="displayName"
+                        type="text"
+                        className="pf-c-form-control"
+                        onChange={this.handleChange}
+                        value={this.state.displayName || ''}
+                      />
+                    </div>
+                  </div>
+                )}
+                {this.props.createProject && (
+                  <div className="form-group">
+                    <label htmlFor="input-description" className="control-label">
+                      {t('public~Description')}
+                    </label>
+                    <div className="modal-body__field">
+                      <textarea
+                        id="input-description"
+                        name="description"
+                        className="pf-c-form-control"
+                        onChange={this.handleChange}
+                        value={this.state.description || ''}
+                      />
+                    </div>
+                  </div>
+                )}
+                {!this.props.createProject && (
+                  <div className="form-group">
+                    <label htmlFor="tags-input" className="control-label">
+                      {t('public~Labels')}
+                    </label>
+                    <div className="modal-body__field">
+                      <SelectorInput
+                        labelClassName="co-text-namespace"
+                        onChange={this.onLabels}
+                        tags={[]}
+                      />
+                    </div>
+                  </div>
+                )}
+                {!this.props.createProject && (
+                  <div className="form-group">
+                    <label htmlFor="network-policy" className="control-label">
+                      {t('public~Default Network Policy')}
```

Comment:
I think this needs to be sentence case too.

```suggestion
                      {t('public~Default network policy')}
```